homeassistant/components/screenlogic/services.py

```diff
@@ -25,6 +31,12 @@
     },
 )
 
+TURN_ON_SUPER_CHLOR_SCHEMA = cv.make_entity_service_schema(
+    {
+        vol.Optional(ATTR_RUNTIME): vol.Clamp(min=MIN_RUNTIME, max=MAX_RUNTIME),
```

Comment:
I'd put the default here instead so you can take it out below

```suggestion
        vol.Optional(ATTR_RUNTIME, default=24): vol.Clamp(min=MIN_RUNTIME, max=MAX_RUNTIME),
```